examples/24a_multicall_busybox.rs

```diff
@@ -0,0 +1,60 @@
+use std::{
+    env::args_os,
+    fs::{hard_link, read_link},
+    path::{Path, PathBuf},
+    process::exit,
+};
+
+use clap::{App, AppSettings, Arg};
+
+fn main() {
+    let app = App::new("busybox")
+        .setting(AppSettings::ArgRequiredElseHelp)
+        .setting(AppSettings::Multicall)
+        .arg(
+            Arg::new("install")
+                .long("install")
+                .about("Install hardlinks for all subcommands in path")
+                .exclusive(true)
+                .takes_value(true)
+                .default_missing_value("/usr/local/bin")
+                .use_delimiter(false),
```

Comment:
This doesn't match the derive example.  Here, its `--install` while in the derive its `install`?

Is this a case of https://github.com/clap-rs/clap/issues/2255#issuecomment-934593346?

Reply:
I wanted to have the derive example also do `--install` to match how busybox handles it but I couldn't get it to work since `SubcommandRequiredElseHelp` appears to be emit unconditionally.
I didn't get as far as trying to mark `--install` as exclusive.

Reply:
That works, I think I was missing that applet has to be an `Option<Applets>`, because obviously it has to if none were given.

Reply:
https://github.com/clap-rs/clap/pull/2820 can possibly make it better in the future, if `install` needs to be required